### DISCUSSION TITLE
Raspberry Pi profiles removal

### DISCUSSION THREAD

**Discussion by salvatore-coppola** [eclipse-kura/kura#5091] [Polls]
Hello Eclipse Kura community,

in effort to simplify and unify the various Eclipse Kura installers and related profiles, we plan to remove the raspberry-pi specific profiles.
This means that the following profiles will be removed with the next version of Kura (5.5.0):

- `raspberry-pi`
- `raspberry-pi-arm64`,
- `raspberry-pi-nn`
- `raspberry-pi-arm64-nn`

The Raspberry Pi platorm will continue to be supported via the `generic` profiles.
The generic profiles offer all the functionality currently found in the specific ones with the exception of the GPIO configuration, which will be added in the documentation.

A further enhancement of the Kura installer may be made in the future to add an automatic GPIO configuration process.

The generic profiles also offer the new, faster and more stable NetworManager-based networking.

What do you think?

**Comment by mattdibi** [comment]
Addressed in #5095
